src/styles/cookbook.css

```
@@ -0,0 +1,199 @@
+  --cb-burgundy-deep: #6E3226;
+  --cb-gold: #C9A24A;
+  --cb-gold-muted: #B08A44;
+  --cb-warm: #8A7654;
+  --cb-card: #FCF7EA;
+  --cb-tile: #F4E9CE;
+  --cb-rule: #D9C49A;
+  --cb-green: #5E7150;
+  --cb-cream: #F7EED6;
+
+  /* faces */
+  --cb-serif: "Cormorant Garamond", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)